src/renderer/components/BouncingSlippiLogo.tsx

```diff
@@ -23,13 +28,18 @@ const Outer = styled.div<{
 
 const Logo = styled.div<{
   size: string;
+  roll: boolean;
```

Comment:
```suggestion
  animationState: "running" | "paused";
```
Maybe we can just pass the animation state in?